packages/target-decisioning-engine/src/constants.js

```diff
@@ -10,6 +10,8 @@ export const DEFAULT_MAXIMUM_WAIT_READY = -1; // default is to wait indefinitely
 export const NUM_FETCH_RETRIES = 10;
 export const SUPPORTED_ARTIFACT_MAJOR_VERSION = 1;
 export const LOG_PREFIX = "LD";
+export const UNKNOWN_IP_ADDRESS = "unknownIpAddress";
```

Comment:
Move this to target-tools constants so it can be used in both the node sdk and at.js without duplication.

Reply:
Done